deploy/operator/utils.sh

```diff
@@ -155,3 +155,26 @@ function nth_ip() {
 
   python -c "from ansible_collections.ansible.netcommon.plugins.filter import ipaddr; print(ipaddr.nthhost('"$network"', $idx))"
 }
+
+function retry() {
+    attempts="${1:-5}"
+    interval="${2:-1}"
```

Comment:
iiuc, usage of optional positional arguments might be weird seeing, because you cannot easily understand what this means:
```
retry 5 oc annotate project openshift-local-storage openshift.io/node-selector='' --overwrite=true  # will it take "oc" as the interval?
```

Reply:
yeah, I was wondering too. I'll use getopts instead. Thanks!

Reply:
Done!